src/Engine/Application.cpp

```diff
@@ -36,57 +44,141 @@ namespace gp1 {
 		m_Mesh->m_Indices.push_back(2);
 		m_Mesh->m_Indices.push_back(1);
 
-		m_Material->SetShader(Shader::GetShader("shader"));
+		m_Material->SetShader(renderer::shader::Shader::GetShader("shader"));
```

Comment:
Should maybe rename the shader to "shader_mesh_default" or something along those lines

Reply:
Sure, though it was only supposed to be a test shader.